docs/download.md

```diff
@@ -52,28 +52,27 @@ do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or late
 need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 22.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 22.10.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 22.12.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.10.0.jar.asc rapids-4-spark_2.12-22.10.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.12.0.jar.asc rapids-4-spark_2.12-22.12.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Dataproc qualification, profiling, bootstrap and diagnostic tool
-* Databricks custom docker container
-* AQE support on Databricks
-* MultiThreaded Shuffle feature
-* Binary write support for parquet
-* Cast binary to string  
-* Hive parquet table write support
+* Deliver z-ordering capability on Databricks
+* Enable DPP on Databricks 10.4
+* Improve robust for workloads which has high compression ratio input datasets
+* Support Delta Lake batch writes on Databricks
+* Partially support HiveTableScanExec on GPU
+* Support mapInArrow
+* Enable tiered projections for more expressions to optimize performance   
 * Qualification and Profiling tool:
-  * Print Databricks cluster/job information
-  * AutoTuner for Profiling tool
+  * Support cost estimations for Dataproc 1.5 and Dataproc2.x
```

Comment:
Can you add a line for the new user tools Git repo?  Example:

`* Added new Github repo for user tools functionality: https://github.com/NVIDIA/spark-rapids-tools/tree/dev/user_tools`

Reply:
added